script/apps/Aries/Creator/Game/Entity/EntityCamera.lua

```diff
@@ -165,12 +165,14 @@ function Entity:SaveCurrentCameraSetting()
 end
 
 function Entity:RestoreCameraSetting()
-	local settings = self:GetRestoreCamSettings();
-	if(settings.eye_dist) then
-		if(CameraController.IsFPSView()~=settings.is_fps) then
-			CameraController.ToggleCamera(settings.is_fps);
+	if not (GameLogic.GetFilters():apply_filters("movie_restore_camera", self)) then
```

Comment:
if you add a filter for paracraft, add doc to D:\lxzsrc\ParaEngine\ParaWorld\script\apps\Aries\Creator\Game\ParacraftFilters.md